spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  context 'Validation' do
+    let(:recipe) { Recipe.new }
+    let(:user) { User.create(name: 'John', email: 'user@gmail.com', password: 'asdf@123') }
+
+    it 'case: Error' do
+
+      # check the error
+      expect { recipe.save! }.to raise_error(ActiveRecord::RecordInvalid)
+    end
+
+    it 'case: Valid' do
+      recipe.name = 'name_of_recipe'
+      recipe.description = 'some description for the recipe'
+      recipe.user = user
```

Comment:
- [ ] Here again, a test is not passing.

![model-test](https://github.com/theresetuyi/Ruby-on-Rails-group-project/assets/53124548/8df2dab7-32d8-4ffc-b7de-a0e061b14bbf)